### ISSUE TITLE
Investigation: Does CommitBuilder need doing now (and for when, what is required)

### ISSUE THREAD

**Issue by BovineOx** [specklesystems/speckle-sharp#2940]
**User Story**
**As a** Speckle Engineer
**I would like** to understand, for ETABS, what is involved with adopting the CommitBuilder pattern used elsewhere
**In order to** assess how to deliver this and whether it is a blocker for future ETABS work

The output of this investigation should be a new ticket detailing what is required and a recommendation and reasoning, on this ticket and socialised, as to whether that new ticket should be done first.

**Comment by bimgeek** [comment]
Closing this ticket as we are moving to Jira to reduce noise. You can access it there.
